Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+            sh:name "environmentConditions"@en ;
+            sh:order 0 ;
+            sh:path scenario:environmentConditions ] ;
+    sh:targetClass scenario:Environmental .
+
+scenario:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "ASAM OpenSCENARIO" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("ASAM OpenSCENARIO") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path scenario:type ],
+        [ skos:example "1.5" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of data format"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of version failed!"@en ;
+            sh:name "version"@en ;
+            sh:order 1 ;
+            sh:path scenario:version ] ;
+    sh:targetClass scenario:Format .
+
+scenario:GeneralShape a sh:NodeShape ;
+    sh:property [ skos:example "Functional, Logical, Concrete" ;
+            sh:datatype xsd:string ;
+            sh:description "Pegasus type of scenario"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path scenario:type ],
+        [ skos:example "cut-in, cut-out" ;
+            sh:datatype xsd:string ;
+            sh:description "Maneuver type in the sense of a functional scenario"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of maneuver failed!"@en ;
+            sh:name "maneuver"@en ;
+            sh:order 1 ;
+            sh:path scenario:maneuver ],
+        [ skos:example "2009-06-30T18:30:00+02:00	" ;
+            sh:datatype xsd:dateTime ;
+            sh:description "Time of the scenario if applicaple. Either time of recording or if synthetic the time it happens."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of timeDate failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "timeDate"@en ;
+            sh:order 2 ;
+            sh:path scenario:timeDate ],
+        [ skos:example "criticality analysis, testing, specific component test (e.g. lateral movement control)" ;
+            sh:datatype xsd:string ;
+            sh:description "Purpose of this scenario."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of aim failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "aim"@en ;
+            sh:order 4 ;
+            sh:path scenario:aim ],
+        [ skos:example "global positions, parameter declaraions..." ;
+            sh:datatype xsd:string ;
+            sh:description "Use of the functionalities from the standard. E.g. which Actions are used."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of usedStandardFunctions failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "usedStandardFunctions"@en ;
```

Comment:
can you add an `sh:in` here?

Reply:
There are a lot of action in OpenSCENARIO. I would suggest to reference to OpenSCENARIO-standard, refering to actions (https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/generated/content/Action.html). Does that get to the essence of this property, @heuerfin?